blocks/video/video.js

```diff
@@ -52,6 +52,23 @@ function embedVimeo(url, replacePlaceholder, autoplay) {
   return temp.children.item(0);
 }
 
+function embedWistia(url, replacePlaceholder, autoplay) {
+  let suffix = '';
+  if (replacePlaceholder || autoplay) {
+    const suffixParams = {
+      autoplay: '1',
+      background: autoplay ? '1' : '0',
+    };
+    suffix = `?${Object.entries(suffixParams).map(([k, v]) => `${k}=${encodeURIComponent(v)}`).join('&')}`;
+  }
+  const temp = document.createElement('div');
+  temp.innerHTML = `<div style="left: 0; width: 100%; height: 0; position: relative; padding-bottom: 56.25%;">
```

Comment:
Any reason we can't use a css class here instead of inline styles?

Reply:
I agree with @seanathero here @MarquiseRosier. Easy change.

Reply:
Plus it'll give you a chance to size the video player fill up the width of the content area.

like this:
![image](https://github.com/user-attachments/assets/547664c3-a174-4d1a-95f3-ae67f238b2fd)